src/main/java/org/broadinstitute/dsm/model/birch/TestBostonResult.java

```diff
@@ -0,0 +1,14 @@
+package org.broadinstitute.dsm.model.birch;
+
+import lombok.Data;
+
+@Data
+public class TestBostonResult {
+    private String orderMessageId;
+    private String institutionId;
+    private String sampleId;
```

Comment:
The name `sampleId` seems a bit foreign to how DSM/Pepper usually communicate. This is the `kitRequestId` right?